developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Move on Aptos"
+slug: "move"
+sidebar_position: 0
+---
+
+# Move on Aptos
+
+The Aptos blockchain consists of validator nodes that run a consensus protocol that agrees upon the ordering of transactions and their output when executed on the MoveVM. Each validator node translates transactions along with the current blockchain ledger state as input into the VM, which processes this input to produce a changeset or storage delta as output. Once consensus agrees and commits to the output, it becomes publicly visible. In this guide, we will introduce you to core Move concepts and how they apply to developing on Aptos.
```

Comment:
These sentences are all run on sentences and full of lots of superfluous and unnecessary words.  Fix it to:

The Aptos blockchain consists of Validator nodes that run a consensus protocol.  The consensus protocol agrees upon the ordering of transactions and their execution output on the Move Virtual Machine (MoveVM). Each Validator node translates transactions with the current blockchain ledger state as input into the MoveVM.  The MoveVM processes the transactions to produce a changeset or storage delta.  Once consensus agrees and commits the output, it becomes publicly visible.  This guide will introduce you to core Move concepts and how they apply to developing on Aptos.